stacks/ucan-invocation-stack.js

```diff
@@ -28,29 +27,33 @@ export function UcanInvocationStack({ stack, app }) {
   setupSentry(app, stack)
 
   // Get dependent stack references
-  const { eventBus } = use(BusStack)
   const { carparkBucket } = use(CarparkStack)
   const { adminMetricsTable, spaceMetricsTable } = use(UploadDbStack)
 
-  const ucanBucket = new Bucket(stack, 'ucan-store', {
+  const workflowBucket = new Bucket(stack, 'workflow-store', {
     cors: true,
     cdk: {
-      bucket: getBucketConfig('ucan-store', app.stage)
+      bucket: getBucketConfig('workflow-store', app.stage)
     }
   })
-
-  // Trigger ucan store events when a CAR is put into the bucket.
-  const ucanPutEventConsumer = new Function(stack, 'ucan-consumer', {
-    environment: {
-      EVENT_BUS_ARN: eventBus.eventBusArn,
-    },
-    permissions: [eventBus],
-    handler: 'functions/ucan-bucket-event.ucanBucketConsumer',
+  const invocationBucket = new Bucket(stack, 'invocation-store', {
+    cors: true,
+    cdk: {
+      bucket: getBucketConfig('invocation-store', app.stage)
+    }
   })
-  ucanBucket.addNotifications(stack, {
-    newCarPut: {
-      function: ucanPutEventConsumer,
-      events: ['object_created_put'],
+  const taskBucket = new Bucket(stack, 'task-store', {
+    cors: true,
+    cdk: {
+      bucket: getBucketConfig('task-store', app.stage)
+    }
+  })
+
+  // TODO: keep for historical content that we might want to process
```

Comment:
Does this definitely keep it around? The act of instantiating a class registers it?

Reply:
Yes, it keeps in Cloudformation. in reality we could even remove it... In prod/staging it won't get deleted per https://github.com/web3-storage/w3infra/blob/main/stacks/config.js#L38

But better to keep it around for now to hook up consumers if we want to still read from there. Otherwise, we will need to hook it up with ARN id